### ISSUE TITLE
Reduced circulating growth hormone concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7586]
**Preferred term label:**
Reduced circulating growth hormone concentration

**Synonyms**

**Definition (free text, please give PubMed ID)**
Concentration of growth hormone in the blood circulation below normal limits.

Currently, [Decreased response to growth hormone stimulation test](https://hpo.jax.org/app/browse/term/HP:0000824) is in the wrong place and should be in functional hierarchy.
Review annotations.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 262400 
PMID: 34470639   1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
